int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jReadOnlyIntegrationTest.java

```diff
@@ -851,7 +851,7 @@ public void testListPlugins() throws IOException {
                 .getPlugins();
 
         assertNotNull(plugins);
-        assertThat(plugins, hasSize(10));
+        assertThat(plugins, hasSize(11));
```

Comment:
@mialbu, out of curiosity, what's the new plugin that makes this number increment? I am badly informed about the changes in 3.6.0

Reply:
> @mialbu, out of curiosity, what's the new plugin that makes this number increment? I am badly informed about the changes in 3.6.0

[`StorageDumper`](https://github.com/neo-project/neo-modules/pull/790)